week-02/assessment/gomoku/src/main/java/learn/gomoku/game/Hud.java

```diff
@@ -0,0 +1,117 @@
+package learn.gomoku.game;
+
+import learn.gomoku.App;
+import learn.gomoku.players.HumanPlayer;
+import learn.gomoku.players.Player;
+import learn.gomoku.players.RandomPlayer;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Hud {
+    static Scanner scn =  new Scanner(System.in);
+
+    /**
+     * This function asks for integers from the user, and uses the pair of integers to return a stone in the spot chosen
+     * @param gomoku
+     * @return stone
+     */
+    public static Stone getNextMove(Gomoku gomoku) {
+        Stone output;
+
+        //System.out.println(gomoku.getCurrent().getName()+"'s Turn");
+        System.out.print("What is the x (col) coordinate of your next move?(1-15): ");
+        int col = Integer.parseInt(scn.nextLine())-1;
+        System.out.print("What is the y (row) coordinate of your next move?(1-15): ");
+        int row = Integer.parseInt(scn.nextLine())-1;
+        return new Stone(row, col, gomoku.isBlacksTurn());
+
+    }
+
+    /**
+     *This function displays a stylized Gomoku board wherein it creates an array
+     * looks through the stones in the list of stones and assigns them to their spot on the array
+     * prints out 1-15 in the rows and columns
+     * and creates the board marking W for white and B for black
+     * @param stones
+     */
+    public static void displayBoard(List<Stone> stones) {
+        int[][] stoneArray = new int[15][15];
```

Comment:
```suggestion
        int[][] stoneArray = new int[Gomoku.WIDTH][Gomoku.WIDTH];
```

Reply:
All of your hard-coded, literal `15` values should be replaced with `Gomoku.WIDTH`.